ui/src/app/utils/index.tsx

```diff
@@ -45,21 +47,13 @@ export function isBlank(toTest: String): boolean {
  * minimum required level to do most things in the Workbench app (outside of
  * local/test development).
  */
-export function hasRegisteredAccess(access: DataAccessLevel): boolean {
+export function hasRegisteredAccessFetch(access: DataAccessLevel): boolean {
```

Comment:
nit: this doesn't need to be called `hasRegisteredAccessFetch` anymore in the absence of `hasRegisteredAccess`